woocommerce/woocommerce
# Update changelogger to 3.3.0 to support PR number capturing with merge

### All Submissions:






### Changes proposed in this Pull Request:



This PR updates the change logger version to support getting PR numbers automatically, even while using the merge strategy.

Note that while other packages also include the change logger, I am limiting the update to only WooCommerce to keep the changes small and testable.






### How to test the changes in this Pull Request:



1. Checkout this branch and run composer install for woocommerce directory.
1. Merge any open PR, for example, #35897 using the default GitHub message and setting by running `git merge fix/35831 --no-ff -m'Merge pull request #35897 from fix/35831'`
2. Merge another open PR, for example #36213, using the GitHub message that we currently use by running `git merge fix/36212 --no-ff -m'Skip custom search for HPOS API queries as its handled already. (#36213)'`
3. Run the changelogger command `pnpm --filter=woocommerce run changelog write --add-pr-num -n -vvv --use-version 7.4`.
4. Check the `NEXT_CHANGELOG.md` file, make sure that all changelogs has the PR numbers, especially 35897 and 36213.



### Other information:




### FOR PR REVIEWER ONLY:
